sunpy/instr/tests/test_aia.py

```diff
@@ -54,3 +55,119 @@ def test_filesave(prep_map):
     np.testing.assert_allclose(prep_map.rotation_matrix, np.identity(2), rtol=1e-5, atol=1e-8)
     # Check level number
     assert load_map.meta['lvl_num'] == 1.5
+
+
+
+
+# def test_aia_read_genx2table():
```

Comment:
That's left over from GSoC. I don't think the tests ever really got started. It needs some...